ecosystem/resources/members/QCRAFT-AutoSchedulQ.toml

```
@@ -0,0 +1,13 @@
+name = "QCRAFT AutoScheduler"
+url = "https://github.com/Qcraft-UEx/QCRAFT-AutoSchedulQ"
+description = "AutoScheduler optimizes quantum circuit execution on IBM Quantum by reducing costs and wait times via task composition and qubit maximization."
```

Thanks! The description is still 142 characters though. I recommend removing the name from the description as the description will appear directly below the name. This brings it to 127 characters.

```suggestion
description = "Optimize quantum circuit execution on IBM Quantum by reducing costs and wait times via task composition and qubit maximization."
```